### DISCUSSION TITLE
PrimeBlocks Figma File vs Figma UI kit

### DISCUSSION THREAD

**Discussion by ricpastori** [primefaces/community#1488] [Figma UI Kit] [answered]
Hi,
My company is considering purchasing a license to use PrimeBlocks. We wanted to know if the "figma file" included in the license matches the UI Kit (https://www.primefaces.org/store/uikit.xhtml)

**Comment by onursenture** [comment, marked as answer]
Hello, at the moment, the PrimeBlocks Figma file only works compatibly with PrimeOne for Figma's 1.0.0 version. Very soon, the upcoming new versions of PrimeBlocks will also work with more current versions of PrimeOne for Figma.

**Reply by ricpastori** [reply]
Thank you @onursenture for explanation and quick response

**Reply by chikko80** [reply]
@onursenture 

I am also completely new to the prime ecosystem but loving it so far and build my first apps. I am also considering buying into primevue, but i am also wondering if the promoted Figma File [https://blocks.primevue.org/pricing](https://blocks.primevue.org/pricing) matches the UI Kit [https://www.primefaces.org/store/uikit.xhtml](https://www.primefaces.org/store/uikit.xhtml), or if these are separate products.

I would really appreciate an updated answer as of today.

Thanks!

**Comment by kleinpetr** [comment]
Hello, is current PrimeBlocks figma compatible with current PrimeOne Figma Kit?

**Reply by onursenture** [reply]
Hello, It is not currently compatible but will be compatible in the next version. We are planning an update very soon with ~30 new blocks and an updated Figma file. Stay tuned.
